x-pack/plugins/ml/public/application/data_frame_analytics/pages/analytics_exploration/components/exploration_results_table/exploration_results_table.tsx

```diff
@@ -55,13 +56,24 @@ interface Props {
   indexPattern: IndexPattern;
   jobConfig: DataFrameAnalyticsConfig;
   jobStatus?: DATA_FRAME_TASK_STATE;
+  needsDestIndexPattern: boolean;
   setEvaluateSearchQuery: React.Dispatch<React.SetStateAction<object>>;
   title: string;
 }
 
 export const ExplorationResultsTable: FC<Props> = React.memo(
-  ({ indexPattern, jobConfig, jobStatus, setEvaluateSearchQuery, title }) => {
+  ({
+    indexPattern,
+    jobConfig,
+    jobStatus,
+    needsDestIndexPattern,
+    setEvaluateSearchQuery,
+    title,
+  }) => {
     const [searchQuery, setSearchQuery] = useState<SavedSearchQuery>(defaultSearchQuery);
+    const destIndex = Array.isArray(jobConfig.dest.index)
```

Comment:
Will `dest.index` ever be an array? Looking at the [docs](https://www.elastic.co/guide/en/elasticsearch/reference/7.8/put-dfanalytics.html) doesn't it have to be a string?

Reply:
You're right - according to the docs, it will always be a string. Updated to take that into account in b7c65a8d71b075768b2c7d4434c260087de30c8a